### ISSUE TITLE
Unexpected error for "of <selector>" syntax

### ISSUE THREAD

**Issue by htunnicliff** [styled-components/typescript-styled-plugin#18]
The **:nth-last-child()** psudeo-class supports an optional selector argument: the [`of <selector>` syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-last-child#the_of_selector_syntax). 

Here is an example:

```tsx
export const Example = styled.div`
  & > *:nth-last-child(-n + 3 of li.important) {
    padding-bottom: 10px;
  }
`;
```

When using this syntax, the following unexpected errors occur:

At the `&` character: 
> } expected [ts-styled-plugin(9999)]

At the closing `}` character: 
> at-rule or selector expected [ts-styled-plugin(9999)]

### Expected Behavior

No errors occur since syntax is valid.
